### DISCUSSION TITLE
Show searching progress icon

### DISCUSSION THREAD

**Discussion by stioboranu** [microsoft-search/pnp-modern-search#2499] [Ideas]
Our colleagues are frustrated because when performing a search they are not able to see that the process is still in progress, all they can see is a freezing Search Results Web Part that unfreeze after the results are returned. Its also frustrating because sometimes you can not know if the filter was applied.

In order to make the process more user friendly we would like to suggest an search progress icon in the Search Results Web Part when the search is performed from the Search Bar Web Part or when a filter is applied from the Search Filters Web Part.

**Comment by wobba** [comment]
There is a shimmer and e.g. the list template has a spinner on the page. Can you clarify more what you refer to?

**Reply by stioboranu** [reply]
Yes, there is a spinner on the first load but we would like for it to appear after adding filters from the Search Filters Web Part. The shimmer is not intuitive enough  for our end users. A boolean setting (see first pic) to always enable the spinner at all types of loading will suffice.
![image](https://user-images.githubusercontent.com/45437451/199501552-8ed9ef15-4f16-4f49-8dce-6a7fd612a6c6.png)
In order to make this one always true on loading.
```
if (this.state.renderedOnce) {

                renderOverlay = <div data-ui-test-id={TestConstants.SearchResultsLoadingOverlay}>
                    <Overlay isDarkThemed={false} className={styles.overlay}>
                        <Spinner size={SpinnerSize.medium} />
                    </Overlay>
                </div>;

            }
```

**Reply by supreet-bajaj** [reply]
Hi @stioboranu,

Did you find any solution to show loading icon on every search with its filters?
I can't see any layout options as well when editing search results web part.

<img width="187" alt="image" src="https://user-images.githubusercontent.com/25117777/201215699-c7c09787-6ce5-4660-aba4-6e0efc487cf4.png">

Thanks,
Supreet Bajaj

**Reply by stioboranu** [reply]
Hi @supreet-bajaj,
There is none right now, this is a request for future releases. I have encountered the same difficulties in order to make visible the spinner at any search but maybe the contributors will add this feature in a new version. Lets hope it will be accomplished.

Have a nice day,
Alexandru Stioboranu

**Reply by supreet-bajaj** [reply]
thanks @stioboranu  for your reply.
